moebot_bot/util/gifmaker.go

```diff
@@ -0,0 +1,119 @@
+package util
+
+import (
+	"bytes"
+	"image"
+	"image/color"
+	"image/gif"
+	"strings"
+
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/basicfont"
+	"golang.org/x/image/math/fixed"
+)
+
+const lineSpace int = 16
```

Comment:
Was `go fmt` run on these files? I'm not sure if it catches these, but if we've got a block of constants we should put them into a `const()` block
```
const (
    lineSpace int = 16
    xBorder int = 30
    yBorder int = 14
    maxWidth int = 500
)
```

Reply:
I ran goreturns (which should in turn call goimports and gofmt)  on the file but it didn't pick up those problems.  
I made sure to fix them manually.